2022-03-22 14:53:20,890 INFO    MainThread:91914 [wandb_setup.py:_flush():75] Loading settings from /Users/AFischer/.config/wandb/settings
2022-03-22 14:53:20,890 INFO    MainThread:91914 [wandb_setup.py:_flush():75] Loading settings from /Users/AFischer/PycharmProjects/cocoon-project/docs/wandb/settings
2022-03-22 14:53:20,890 INFO    MainThread:91914 [wandb_setup.py:_flush():75] Loading settings from environment variables: {}
2022-03-22 14:53:20,890 INFO    MainThread:91914 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2022-03-22 14:53:20,890 INFO    MainThread:91914 [wandb_init.py:_log_setup():405] Logging user logs to /Users/AFischer/PycharmProjects/cocoon-project/docs/wandb/run-20220322_145320-34dem8qe/logs/debug.log
2022-03-22 14:53:20,890 INFO    MainThread:91914 [wandb_init.py:_log_setup():406] Logging internal logs to /Users/AFischer/PycharmProjects/cocoon-project/docs/wandb/run-20220322_145320-34dem8qe/logs/debug-internal.log
2022-03-22 14:53:20,890 INFO    MainThread:91914 [wandb_init.py:init():439] calling init triggers
2022-03-22 14:53:20,890 INFO    MainThread:91914 [wandb_init.py:init():442] wandb.init called with sweep_config: {}
config: {}
2022-03-22 14:53:20,891 INFO    MainThread:91914 [wandb_init.py:init():458] re-initializing run, found existing run on stack: to4npspg
2022-03-22 14:53:20,892 INFO    MainThread:91914 [wandb_run.py:finish():1539] finishing run afischer/tcn_samp_en-project/to4npspg
2022-03-22 14:53:20,893 INFO    MainThread:91914 [jupyter.py:save_history():429] not saving jupyter history
2022-03-22 14:53:20,893 INFO    MainThread:91914 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-03-22 14:53:20,893 INFO    MainThread:91914 [wandb_init.py:_jupyter_teardown():337] cleaning up jupyter logic
2022-03-22 14:53:20,893 INFO    MainThread:91914 [wandb_run.py:_atexit_cleanup():1702] got exitcode: 0
2022-03-22 14:53:20,896 INFO    MainThread:91914 [wandb_run.py:_restore():1674] restore
2022-03-22 14:53:21,625 INFO    MainThread:91914 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 825
  total_bytes: 825
}

2022-03-22 14:53:21,895 INFO    MainThread:91914 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 825
  total_bytes: 825
}

2022-03-22 14:53:22,634 INFO    MainThread:91914 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 825
  total_bytes: 100899
}

2022-03-22 14:53:22,744 INFO    MainThread:91914 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 825
  total_bytes: 100899
}

2022-03-22 14:53:22,856 INFO    MainThread:91914 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 825
  total_bytes: 100899
}

2022-03-22 14:53:22,962 INFO    MainThread:91914 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 94872
  total_bytes: 100899
}

2022-03-22 14:53:23,075 INFO    MainThread:91914 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 100899
  total_bytes: 100899
}

2022-03-22 14:53:23,180 INFO    MainThread:91914 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 100899
  total_bytes: 100899
}

2022-03-22 14:53:23,292 INFO    MainThread:91914 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 100899
  total_bytes: 100899
}

2022-03-22 14:53:23,407 INFO    MainThread:91914 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 100899
  total_bytes: 100899
}

2022-03-22 14:53:23,517 INFO    MainThread:91914 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 100899
  total_bytes: 100899
}

2022-03-22 14:53:23,633 INFO    MainThread:91914 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 100899
  total_bytes: 100899
}

2022-03-22 14:53:24,829 INFO    MainThread:91914 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 100899
  total_bytes: 100899
}

2022-03-22 14:53:25,864 INFO    MainThread:91914 [wandb_run.py:_on_finish():1831] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 100899
  total_bytes: 100899
}
local_info {
}

2022-03-22 14:53:27,032 INFO    MainThread:91914 [wandb_run.py:_footer_history_summary_info():2865] rendering history
2022-03-22 14:53:27,034 INFO    MainThread:91914 [wandb_run.py:_footer_history_summary_info():2894] rendering summary
2022-03-22 14:53:27,036 INFO    MainThread:91914 [wandb_run.py:_footer_sync_info():2822] logging synced files
2022-03-22 14:53:27,038 INFO    MainThread:91914 [wandb_init.py:init():492] starting backend
2022-03-22 14:53:27,038 INFO    MainThread:91914 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2022-03-22 14:53:27,041 INFO    MainThread:91914 [backend.py:ensure_launched():219] starting backend process...
2022-03-22 14:53:27,060 INFO    MainThread:91914 [backend.py:ensure_launched():224] started backend process with pid: 93252
2022-03-22 14:53:27,062 INFO    MainThread:91914 [wandb_init.py:init():501] backend started and connected
2022-03-22 14:53:27,082 INFO    MainThread:91914 [wandb_run.py:_label_probe_notebook():947] probe notebook
2022-03-22 14:53:27,082 INFO    MainThread:91914 [wandb_run.py:_label_probe_notebook():957] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-03-22 14:53:27,082 INFO    MainThread:91914 [wandb_init.py:init():565] updated telemetry
2022-03-22 14:53:27,113 INFO    MainThread:91914 [wandb_init.py:init():596] communicating run to backend with 30 second timeout
2022-03-22 14:53:28,640 INFO    MainThread:91914 [wandb_run.py:_on_init():1759] communicating current version
2022-03-22 14:53:28,712 INFO    MainThread:91914 [wandb_run.py:_on_init():1763] got version response 
2022-03-22 14:53:28,712 INFO    MainThread:91914 [wandb_init.py:init():625] starting run threads in backend
2022-03-22 14:53:30,817 INFO    MainThread:91914 [wandb_run.py:_console_start():1733] atexit reg
2022-03-22 14:53:30,819 INFO    MainThread:91914 [wandb_run.py:_redirect():1606] redirect: SettingsConsole.WRAP
2022-03-22 14:53:30,819 INFO    MainThread:91914 [wandb_run.py:_redirect():1643] Wrapping output streams.
2022-03-22 14:53:30,819 INFO    MainThread:91914 [wandb_run.py:_redirect():1667] Redirects installed.
2022-03-22 14:53:30,819 INFO    MainThread:91914 [wandb_init.py:init():664] run started, returning control to user process
2022-03-22 14:53:30,820 INFO    MainThread:91914 [wandb_watch.py:watch():47] Watching
2022-03-22 14:58:44,060 INFO    MainThread:91914 [wandb_config.py:__setitem__():153] config set batch_size = 60 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x7fc54f6a21c0>>
2022-03-22 14:58:44,061 INFO    MainThread:91914 [wandb_run.py:_config_callback():992] config_cb batch_size 60 None
2022-03-22 14:58:44,061 INFO    MainThread:91914 [wandb_config.py:__setitem__():153] config set epochs = 30 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x7fc54f6a21c0>>
2022-03-22 14:58:44,061 INFO    MainThread:91914 [wandb_run.py:_config_callback():992] config_cb epochs 30 None
2022-03-22 14:58:44,061 INFO    MainThread:91914 [wandb_config.py:__setitem__():153] config set lr = 1.640720398047456e-05 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x7fc54f6a21c0>>
2022-03-22 14:58:44,061 INFO    MainThread:91914 [wandb_run.py:_config_callback():992] config_cb lr 1.640720398047456e-05 None
2022-03-22 14:58:44,061 INFO    MainThread:91914 [wandb_config.py:__setitem__():153] config set no_cuda = False - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x7fc54f6a21c0>>
2022-03-22 14:58:44,061 INFO    MainThread:91914 [wandb_run.py:_config_callback():992] config_cb no_cuda False None
2022-03-22 14:58:44,061 INFO    MainThread:91914 [wandb_config.py:__setitem__():153] config set seed = 42 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x7fc54f6a21c0>>
2022-03-22 14:58:44,061 INFO    MainThread:91914 [wandb_run.py:_config_callback():992] config_cb seed 42 None
2022-03-22 15:01:37,192 INFO    MainThread:91914 [wandb_setup.py:_flush():75] Loading settings from /Users/AFischer/.config/wandb/settings
2022-03-22 15:01:37,193 INFO    MainThread:91914 [wandb_setup.py:_flush():75] Loading settings from /Users/AFischer/PycharmProjects/cocoon-project/docs/wandb/settings
2022-03-22 15:01:37,193 INFO    MainThread:91914 [wandb_setup.py:_flush():75] Loading settings from environment variables: {}
2022-03-22 15:01:37,193 INFO    MainThread:91914 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2022-03-22 15:01:37,193 INFO    MainThread:91914 [wandb_init.py:_log_setup():405] Logging user logs to /Users/AFischer/PycharmProjects/cocoon-project/docs/wandb/run-20220322_150137-3l0mpbn9/logs/debug.log
2022-03-22 15:01:37,193 INFO    MainThread:91914 [wandb_init.py:_log_setup():406] Logging internal logs to /Users/AFischer/PycharmProjects/cocoon-project/docs/wandb/run-20220322_150137-3l0mpbn9/logs/debug-internal.log
2022-03-22 15:01:37,193 INFO    MainThread:91914 [wandb_init.py:_jupyter_setup():355] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7fc6ac2d3880>
2022-03-22 15:01:37,195 INFO    MainThread:91914 [wandb_init.py:init():439] calling init triggers
2022-03-22 15:01:37,195 INFO    MainThread:91914 [wandb_init.py:init():442] wandb.init called with sweep_config: {}
config: {}
2022-03-22 15:01:37,195 INFO    MainThread:91914 [wandb_init.py:init():458] re-initializing run, found existing run on stack: 34dem8qe
2022-03-22 15:01:37,201 INFO    MainThread:91914 [wandb_run.py:finish():1539] finishing run afischer/tcn_samp_en-project/34dem8qe
2022-03-22 15:01:37,201 INFO    MainThread:91914 [wandb_run.py:_atexit_cleanup():1702] got exitcode: 0
2022-03-22 15:01:37,225 INFO    MainThread:91914 [wandb_run.py:_restore():1674] restore
2022-03-22 15:01:37,858 INFO    MainThread:91914 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 825
  total_bytes: 825
}

2022-03-22 15:01:38,077 INFO    MainThread:91914 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 825
  total_bytes: 825
}

2022-03-22 15:01:38,809 INFO    MainThread:91914 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 825
  total_bytes: 158591
}

2022-03-22 15:01:38,918 INFO    MainThread:91914 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 825
  total_bytes: 164358
}

2022-03-22 15:01:39,026 INFO    MainThread:91914 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 825
  total_bytes: 164358
}

2022-03-22 15:01:39,134 INFO    MainThread:91914 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 164358
  total_bytes: 164358
}

2022-03-22 15:01:39,242 INFO    MainThread:91914 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 164358
  total_bytes: 164358
}

2022-03-22 15:01:39,350 INFO    MainThread:91914 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 164358
  total_bytes: 164358
}

2022-03-22 15:01:39,464 INFO    MainThread:91914 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 164358
  total_bytes: 164358
}

2022-03-22 15:01:39,569 INFO    MainThread:91914 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 164358
  total_bytes: 164358
}

2022-03-22 15:01:39,674 INFO    MainThread:91914 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 164358
  total_bytes: 164358
}

2022-03-22 15:01:40,556 INFO    MainThread:91914 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 164358
  total_bytes: 164358
}

2022-03-22 15:01:41,014 INFO    MainThread:91914 [wandb_run.py:_on_finish():1831] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 164358
  total_bytes: 164358
}
local_info {
}

2022-03-22 15:01:42,172 INFO    MainThread:91914 [wandb_run.py:_footer_history_summary_info():2865] rendering history
2022-03-22 15:01:42,175 INFO    MainThread:91914 [wandb_run.py:_footer_history_summary_info():2894] rendering summary
2022-03-22 15:01:42,180 INFO    MainThread:91914 [wandb_run.py:_footer_sync_info():2822] logging synced files
